content/en/apps/tutorials/local-setup.md

```diff
@@ -64,42 +64,42 @@ Once the command is done running, navigate to [https://localhost](https://localh
 
 If you encounter an error `bind: address already in use`, see the [Port Conflicts section]({{< relref "core/guides/docker-setup#port-conflicts" >}}) in our Docker Setup guide.
 
-This CHT instance is empty and has no data in it.  While you're free to explore and add your own data, in step 3 below we'll upload sample data.  Proceed to step 2 to install `medic-conf` which is needed to upload the test data.
+This CHT instance is empty and has no data in it.  While you're free to explore and add your own data, in step 3 below we'll upload sample data.  Proceed to step 2 to install `cht-conf` which is needed to upload the test data.
 
 <br clear="all">
 
  *****
 
-### 2. Install medic-conf
+### 2. Install cht-conf
 
-Using npm and python on your terminal, install medic-conf and pyxform globally using the following commands:
+Using npm and python on your terminal, install cht-conf and pyxform globally using the following commands:
```

Comment:
there's actually [a screen shot](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/#2-install-medic-conf) on this page that we should update too to match the new name!

Reply:
Oh good call!  Switched it for an updated screenshot.